vulnerabilities/unsupported-opcodes.md

```diff
@@ -0,0 +1,51 @@
+# Unsupported Opcodes
+
+EVM-compatible chains, such as zkSync Era, BNB Chain, Polygon, Optimism and Arbitrum implement the Ethereum Virtual Machine (EVM) and its opcodes. However, opcode support can vary across these chains, which can lead to bugs and issues if not considered during smart contract development and deployment.
+
+## PUSH0 Opcode Compatibility
+The `PUSH0` opcode was introduced during the Shanghai hard fork of the Shapella upgrade (Solidity v0.8.20) and is available in certain EVM-compatible chains. However, not all chains have implemented support for this opcode yet.
+
+### Is `PUSH0` supported:
+1. Ethereum: YES
+2. Arbitrum One: YES
+3. Optimism: YES
+4. ...
+
+More chain differences and opcode support can be found on: [evmdiff.com](https://www.evmdiff.com)
+
+You can also check compatibility by running the following command assuming you have Foundry set up:
+
+```bash
+cast call --block 176 --rpc-url $ARBITRUM_RPC_URL --create 0x5f
```

Comment:
I don't think we need to include the --block flag here. It's useful to look at historical support but generally we just want to check for current support